### ISSUE TITLE
bugfix/edit insurance beneficiary

### ISSUE THREAD

**Issue by TuyizereBapt** [serenity-health/roadmap#126]
**Describe the bug**
There is a mismatch between the backend implementation and front-end usage of the API endpoint for editing an insurance beneficiary (https://serenity-health.readme.io/reference/insurance-clients-and-payments#edit-an-insurance-beneficiary)

**Impact**
_What is the impact of this problem_?

**To Reproduce**
1. Go to 'https://demo.provider.serenity.health/clients/25c92ebe-30fd-4cc4-bfcf-5958b77bd7e6/benefactors'
2. Search for a specific beneficiary, and under action menu, click on edit
3. Make modifications and click update beneficiary button (_Side note: insurance provider field and health policy fields could not load data for selection when I was testing. And not all data is pre-filled into the fields_)

**Note that, although an employee can also be an insurance beneficiary with a SELF (independent) relationship, there are other patients who can be insurance beneficiaries with DEPENDENT relationship**

**Expected behavior**
The insurance beneficiary should successfully be updated

**Screenshots/Screencast**
![image](https://user-images.githubusercontent.com/37751827/160933878-058ee11f-84fb-4eab-aad4-736b90c5c611.png)

Request payload. From the request payload below, it is clear that what's being sent is an employee's data
![image](https://user-images.githubusercontent.com/37751827/160931897-c8599d7b-d453-4d26-8912-e31308706df4.png)

What the API expects
![image](https://user-images.githubusercontent.com/37751827/160931959-b671a36a-be56-4970-a04a-841e4ec30ec4.png)

**Additional context**
N/A

**Comment by TuyizereBapt** [comment]
@dexios1 let's look at this tomorrow to get it out of way

**Comment by TuyizereBapt** [comment]
Fixed by: https://github.com/serenity-health/core-systems-api/pull/87

**Comment by chris-dare** [comment]
Hi @TuyizereBapt please take Dadson through the integration of this

**Comment by TuyizereBapt** [comment]
> Hi @TuyizereBapt please take Dadson through the integration of this 

Yes. @pkdadson should let me know any challenges he might face while integrating this. I remember he was the one who raised this issue.
